common/utils/formatters.ts

```diff
@@ -8,9 +8,38 @@ export function combineAndUpper(...args: string[]) {
   return args.reduce((acc, item) => acc.concat(item.toUpperCase()), '');
 }
 
+const toFixed = (num: string, digits: number = 3) => {
```

Comment:
Could use a test.

Reply:
It's tested in `formatNumber` unless you wanted a seperate test for it